commands_base/__init__.py

```diff
@@ -52,8 +54,11 @@ def main(project_name: str):
         ),
     )
 
-    # Invoke the requested command
-    args.func(args)
+    try:
+        # Invoke the requested command
+        args.func(args)
+    except AttributeError:
+        parser.print_help()
```

Comment:
If args.func doesn't exist at this point, doesn't that mean there's a bug, not user error?  Either the user specified an invalid command (which should be caught at parse_args()), or they specified a valid one but the command didn't register a func().  In the former case, we already print_help and exit; in the latter case, it seems like we should fail loudly.

Also... catching all AttributeErrors here seems like it risks masking bugs that raise that exception from within the command.

Reply:
Hopefully the intention is now a bit more clear, good call on not having it be a try/except